sup3r/preprocessing/batch_handling.py

```diff
@@ -127,7 +133,8 @@ def get_coarse_batch(cls, high_res,
         if t_enhance != 1:
             low_res = temporal_coarsening(low_res, t_enhance,
                                           temporal_coarsening_method)
-
+        if smoothing is not None:
+            low_res = gaussian_filter(low_res, smoothing, mode='constant')
```

Comment:
Is this what you're thinking for making the coarsened data look more like coarse WRF outputs with less high-res turbulence?

Reply:
Yeah exactly